vendor/assets/javascripts/data-confirm-modal.js

```diff
@@ -137,6 +137,16 @@
       '</div>'
     );
 
+    // Make sure it's always the top zindex
+    var highest = 1050;
```

Comment:
Why are we choosing `1050` as a magic number? As far as I can see if we start from 1 the algorithm below will work as well